### PR TITLE
Vintage Delight 翻译提交

### PR DESCRIPTION
**Pull request by SieboldiiParlia**

### CODE REVIEW

**Review by SieboldiiParlia — COMMENTED** [review]
无误

File: projects/1.20/assets/vintage-delight/vintagedelight/lang/zh_cn.json [hunk 1]
```diff
@@ -0,0 +1,186 @@
+{
+  "creativetab.vintage_tab": "腌渍选项卡",
+
+  //Blocks
+    "block.vintagedelight.fermenting_jar": "腌制罐",
+    "block.vintagedelight.lush_grass_block": "繁茂草地",
+    "block.vintagedelight.salt_lamp_default": "盐灯",
+    "block.vintagedelight.salt_lamp_white": "白色盐灯",
+    "block.vintagedelight.salt_lamp_orange": "橘色盐灯",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/vintage-delight/vintagedelight/lang/zh_cn.json:9]:
```suggestion
    "block.vintagedelight.salt_lamp_orange": "橙色盐灯",
```
原版术语

File: projects/1.20/assets/vintage-delight/vintagedelight/lang/zh_cn.json [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+  //Items
+    "item.vintagedelight.salt_dust": "盐",
+    "item.vintagedelight.ghostly_chili": "幽灵辣椒",
+    "item.vintagedelight.pad_thai": "泰式炒河粉",
+    "item.vintagedelight.oat": "燕麦",
+    "item.vintagedelight.raw_oats": "生燕麦",
+    "item.vintagedelight.oat_seeds": "燕麦种子",
+    "item.vintagedelight.peanut": "花生",
+    "item.vintagedelight.roasted_peanut": "烤制花生",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/vintage-delight/vintagedelight/lang/zh_cn.json:75]: 要不直接`烤花生`吧

File: projects/1.20/assets/vintage-delight/vintagedelight/lang/zh_cn.json [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+    "item.vintagedelight.chocolate_nut_granola_bar": "巧克力坚果燕麦棒",
+    "item.vintagedelight.fruity_granola_bar": "果味燕麦棒",
+    "item.vintagedelight.deluxe_granola_bar": "豪华燕麦棒",
+    "item.vintagedelight.mason_jar": "梅森罐",
+    "item.vintagedelight.apple_sauce_mason_jar": "苹果汁罐",
+    "item.vintagedelight.apple_sauce_bottle": "苹果汁瓶",
+    "item.vintagedelight.sweet_berry_mason_jar": "甜浆果果酱罐",
+    "item.vintagedelight.sweet_berry_jam_bottle": "甜浆果果酱瓶",
+    "item.vintagedelight.glow_berry_mason_jar": "荧光果果酱罐",
+    "item.vintagedelight.glow_berry_jam_bottle": "荧光果果酱瓶",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/vintage-delight/vintagedelight/lang/zh_cn.json:113]:
```suggestion
    "item.vintagedelight.glow_berry_mason_jar": "发光浆果果酱罐",
    "item.vintagedelight.glow_berry_jam_bottle": "发光浆果果酱瓶",
```
原版名称

File: projects/1.20/assets/vintage-delight/vintagedelight/lang/zh_cn.json [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+    "item.vintagedelight.blue_chefs_hat": "蓝色厨师帽",
+    "item.vintagedelight.brown_chefs_hat": "棕色厨师帽",
+    "item.vintagedelight.cyan_chefs_hat": "青色厨师帽",
+    "item.vintagedelight.gray_chefs_hat": "灰色厨师帽",
+    "item.vintagedelight.green_chefs_hat": "绿色厨师帽",
+    "item.vintagedelight.light_blue_chefs_hat": "淡蓝色厨师帽",
+    "item.vintagedelight.light_gray_chefs_hat": "淡灰色厨师帽",
+    "item.vintagedelight.lime_chefs_hat": "黄绿色厨师帽",
+    "item.vintagedelight.magenta_chefs_hat": "品红色厨师帽",
+    "item.vintagedelight.orange_chefs_hat": "橘色厨师帽",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/vintage-delight/vintagedelight/lang/zh_cn.json:133]:
```suggestion
    "item.vintagedelight.orange_chefs_hat": "橙色厨师帽",
```

File: projects/1.20/assets/vintage-delight/vintagedelight/lang/zh_cn.json [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+    "item.vintagedelight.red_chefs_hat": "红色厨师帽",
+    "item.vintagedelight.white_chefs_hat": "白色厨师帽",
+    "item.vintagedelight.yellow_chefs_hat": "黄色厨师帽",
+    "item.vintagedelight.salt_bucket": "盐桶",
+
+    "item.vintagedelight.salted_cod": "腌鳕鱼",
+    "item.vintagedelight.salted_salmon": "腌鲑鱼",
+    "item.vintagedelight.surstromming": "鲱鱼罐头",
+
+    "item.vintagedelight.pb_j": "花生酱和果酱三明治",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/vintage-delight/vintagedelight/lang/zh_cn.json:145]:
```suggestion
    "item.vintagedelight.pb_j": "花生酱果酱三明治",
```

File: projects/1.20/assets/vintage-delight/vintagedelight/lang/zh_cn.json [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+
+
+  "vintagedelight.advancement.root": "腌渍乐事",
+  "vintagedelight.advancement.root.desc": "来腌制吧!",
+  "vintagedelight.advancement.harvest_salt": "咸咸的",
+  "vintagedelight.advancement.harvest_salt.desc": "收获盐",
+  "vintagedelight.advancement.place_fermenting_jar": "腌渍",
+  "vintagedelight.advancement.place_fermenting_jar.desc": "放置一个腌制罐",
+  "vintagedelight.advancement.place_salt_block": "产盐",
+  "vintagedelight.advancement.place_salt_block.desc": "放置在水源附近以产盐.",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/vintage-delight/vintagedelight/lang/zh_cn.json:168]:
```suggestion
  "vintagedelight.advancement.place_salt_block.desc": "放置在水源附近以产盐。",
```
请注意使用全角标点

File: projects/1.20/assets/vintage-delight/vintagedelight/lang/zh_cn.json [hunk 7]
```diff
@@ -0,0 +1,186 @@
...
+
+
+
+  "item.vintagedelight.mason_jar.tooltip": "可放置",
+  "farmersdelight.tooltip.nut_milk_bottle": "清除1个效果",
+  "gui.jei.category.fermenting.time.seconds": "腌制时间",
+  "vintagedelight.jei.info.salt_block": "放置在水源附近以产盐",
+  "vintagedelight.jei.info.evaporator": "放置在水源附近，以比盐块快得多的速度产盐",
+  "vintagedelight.jei.info.organic_compost": "腌制有机醪泥，用作强效肥料，或用于堆肥",
+  "vintagedelight.jei.info.cheese_wheel": "将 4个奶酪凝乳放入奶酪模具中以制作奶酪轮",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/vintage-delight/vintagedelight/lang/zh_cn.json:157]:
```suggestion
  "vintagedelight.jei.info.cheese_wheel": "将4个奶酪凝乳放入奶酪模具中以制作奶酪轮",
```
多余空格

File: projects/1.20/assets/vintage-delight/vintagedelight/lang/zh_cn.json [hunk 8]
```diff
@@ -0,0 +1,186 @@
...
+  "vintagedelight.advancement.place_evaporator": "蒸发器",
+  "vintagedelight.advancement.place_evaporator.desc": "放置一个被水包围的蒸发器，以比盐块快得多的速度产盐",
+  "vintagedelight.advancement.get_pickled_item": "在腌制中",
+  "vintagedelight.advancement.get_pickled_item.desc": "腌制一次蔬菜",
+  "vintagedelight.advancement.obtain_magic_peanut": "魔力花生!",
+  "vintagedelight.advancement.obtain_magic_peanut.desc": "也许是一只金鹅?",
+  "vintagedelight.advancement.obtain_century_egg": "获得世纪彩蛋",
+  "vintagedelight.advancement.obtain_century_egg.desc": "还没有为公布它做好准备",
+
+  "vintagedelight.advancement.get_jam_item": "粘糊糊'",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/vintage-delight/vintagedelight/lang/zh_cn.json:178]:
```suggestion
  "vintagedelight.advancement.get_jam_item": "黏糊糊",
```
原文`Jammin'`最后的单引号代表省音，是口语中把/ŋ/发成/n/的现象，不保留

File: projects/1.20/assets/vintage-delight/vintagedelight/lang/zh_cn.json [hunk 9]
```diff
@@ -0,0 +1,186 @@
+{
+  "creativetab.vintage_tab": "腌渍选项卡",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/vintage-delight/vintagedelight/lang/zh_cn.json:2]: 这个怪怪的（）
Vintage应当是模组名的一部分，推荐使用`模组名`或者直接`模组名+“选项卡”`

**Review by ChuijkYahus — CHANGES_REQUESTED** [review]
记得改PR名称

**Comment by SieboldiiParlia** [comment]
好的谢谢，改完了重新发次pr吗



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年12月22日(星期五) 下午2:13
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [CFPAOrg/Minecraft-Mod-Language-Package] Vintage Delight 翻译提交 (PR #4022)





 
@ChuijkYahus requested changes on this pull request.
 
记得改PR名称

 
In projects/1.20/assets/vintage-delight/vintagedelight/lang/zh_cn.json:
 &gt; @@ -0,0 +1,186 @@ +{ +  "creativetab.vintage_tab": "腌渍选项卡", + +  //Blocks +    "block.vintagedelight.fermenting_jar": "腌制罐", +    "block.vintagedelight.lush_grass_block": "繁茂草地", +    "block.vintagedelight.salt_lamp_default": "盐灯", +    "block.vintagedelight.salt_lamp_white": "白色盐灯", +    "block.vintagedelight.salt_lamp_orange": "橘色盐灯",  ⬇️ Suggested change -    "block.vintagedelight.salt_lamp_orange": "橘色盐灯", +    "block.vintagedelight.salt_lamp_orange": "橙色盐灯",  
原版术语
 
 
In projects/1.20/assets/vintage-delight/vintagedelight/lang/zh_cn.json:
 &gt; +    "block.vintagedelight.cheese_wheel": "奶酪轮", +    "block.vintagedelight.cheese_mold": "奶酪模具", +    "block.vintagedelight.cheese_pizza": "奶酪披萨", +    "block.vintagedelight.meat_pizza": "肉类披萨", +    "block.vintagedelight.vine_net": "藤网", + + +  //Items +    "item.vintagedelight.salt_dust": "盐", +    "item.vintagedelight.ghostly_chili": "幽灵辣椒", +    "item.vintagedelight.pad_thai": "泰式炒河粉", +    "item.vintagedelight.oat": "燕麦", +    "item.vintagedelight.raw_oats": "生燕麦", +    "item.vintagedelight.oat_seeds": "燕麦种子", +    "item.vintagedelight.peanut": "花生", +    "item.vintagedelight.roasted_peanut": "烤制花生",  
要不直接烤花生吧
 
 
In projects/1.20/assets/vintage-delight/vintagedelight/lang/zh_cn.json:
 &gt; +    "item.vintagedelight.glow_berry_mason_jar": "荧光果果酱罐", +    "item.vintagedelight.glow_berry_jam_bottle": "荧光果果酱瓶",  ⬇️ Suggested change -    "item.vintagedelight.glow_berry_mason_jar": "荧光果果酱罐", -    "item.vintagedelight.glow_berry_jam_bottle": "荧光果果酱瓶", +    "item.vintagedelight.glow_berry_mason_jar": "发光浆果果酱罐", +    "item.vintagedelight.glow_berry_jam_bottle": "发光浆果果酱瓶",  
原版名称
 
 
In projects/1.20/assets/vintage-delight/vintagedelight/lang/zh_cn.json:
 &gt; +    "item.vintagedelight.cheese_slice": "奶酪切片", +    "item.vintagedelight.organic_mash": "有机醪泥", +    "item.vintagedelight.cheese_pizza_slice": "奶酪披萨切片", +    "item.vintagedelight.meat_pizza_slice": "肉类披萨切片", +    "item.vintagedelight.cucumber_salad": "黄瓜沙拉", +    "item.vintagedelight.black_chefs_hat": "黑色厨师帽", +    "item.vintagedelight.blue_chefs_hat": "蓝色厨师帽", +    "item.vintagedelight.brown_chefs_hat": "棕色厨师帽", +    "item.vintagedelight.cyan_chefs_hat": "青色厨师帽", +    "item.vintagedelight.gray_chefs_hat": "灰色厨师帽", +    "item.vintagedelight.green_chefs_hat": "绿色厨师帽", +    "item.vintagedelight.light_blue_chefs_hat": "淡蓝色厨师帽", +    "item.vintagedelight.light_gray_chefs_hat": "淡灰色厨师帽", +    "item.vintagedelight.lime_chefs_hat": "黄绿色厨师帽", +    "item.vintagedelight.magenta_chefs_hat": "品红色厨师帽", +    "item.vintagedelight.orange_chefs_hat": "橘色厨师帽",  ⬇️ Suggested change -    "item.vintagedelight.orange_chefs_hat": "橘色厨师帽", +    "item.vintagedelight.orange_chefs_hat": "橙色厨师帽",  
 
In projects/1.20/assets/vintage-delight/vintagedelight/lang/zh_cn.json:
 &gt; +    "item.vintagedelight.light_gray_chefs_hat": "淡灰色厨师帽", +    "item.vintagedelight.lime_chefs_hat": "黄绿色厨师帽", +    "item.vintagedelight.magenta_chefs_hat": "品红色厨师帽", +    "item.vintagedelight.orange_chefs_hat": "橘色厨师帽", +    "item.vintagedelight.pink_chefs_hat": "粉红色厨师帽", +    "item.vintagedelight.purple_chefs_hat": "紫色厨师帽", +    "item.vintagedelight.red_chefs_hat": "红色厨师帽", +    "item.vintagedelight.white_chefs_hat": "白色厨师帽", +    "item.vintagedelight.yellow_chefs_hat": "黄色厨师帽", +    "item.vintagedelight.salt_bucket": "盐桶", + +    "item.vintagedelight.salted_cod": "腌鳕鱼", +    "item.vintagedelight.salted_salmon": "腌鲑鱼", +    "item.vintagedelight.surstromming": "鲱鱼罐头", + +    "item.vintagedelight.pb_j": "花生酱和果酱三明治",  ⬇️ Suggested change -    "item.vintagedelight.pb_j": "花生酱和果酱三明治", +    "item.vintagedelight.pb_j": "花生酱果酱三明治",  
 
In projects/1.20/assets/vintage-delight/vintagedelight/lang/zh_cn.json:
 &gt; +  "gui.jei.category.fermenting.time.seconds": "腌制时间", +  "vintagedelight.jei.info.salt_block": "放置在水源附近以产盐", +  "vintagedelight.jei.info.evaporator": "放置在水源附近，以比盐块快得多的速度产盐", +  "vintagedelight.jei.info.organic_compost": "腌制有机醪泥，用作强效肥料，或用于堆肥", +  "vintagedelight.jei.info.cheese_wheel": "将 4个奶酪凝乳放入奶酪模具中以制作奶酪轮", + + + +  "vintagedelight.advancement.root": "腌渍乐事", +  "vintagedelight.advancement.root.desc": "来腌制吧!", +  "vintagedelight.advancement.harvest_salt": "咸咸的", +  "vintagedelight.advancement.harvest_salt.desc": "收获盐", +  "vintagedelight.advancement.place_fermenting_jar": "腌渍", +  "vintagedelight.advancement.place_fermenting_jar.desc": "放置一个腌制罐", +  "vintagedelight.advancement.place_salt_block": "产盐", +  "vintagedelight.advancement.place_salt_block.desc": "放置在水源附近以产盐.",  ⬇️ Suggested change -  "vintagedelight.advancement.place_salt_block.desc": "放置在水源附近以产盐.", +  "vintagedelight.advancement.place_salt_block.desc": "放置在水源附近以产盐。",  
请注意使用全角标点
 
 
In projects/1.20/assets/vintage-delight/vintagedelight/lang/zh_cn.json:
 &gt; +    "item.vintagedelight.salted_salmon": "腌鲑鱼", +    "item.vintagedelight.surstromming": "鲱鱼罐头", + +    "item.vintagedelight.pb_j": "花生酱和果酱三明治", +    "item.vintagedelight.cheese_burger": "芝士汉堡", +    "item.vintagedelight.deluxe_burger": "豪华汉堡", + + + +  "item.vintagedelight.mason_jar.tooltip": "可放置", +  "farmersdelight.tooltip.nut_milk_bottle": "清除1个效果", +  "gui.jei.category.fermenting.time.seconds": "腌制时间", +  "vintagedelight.jei.info.salt_block": "放置在水源附近以产盐", +  "vintagedelight.jei.info.evaporator": "放置在水源附近，以比盐块快得多的速度产盐", +  "vintagedelight.jei.info.organic_compost": "腌制有机醪泥，用作强效肥料，或用于堆肥", +  "vintagedelight.jei.info.cheese_wheel": "将 4个奶酪凝乳放入奶酪模具中以制作奶酪轮",  ⬇️ Suggested change -  "vintagedelight.jei.info.cheese_wheel": "将 4个奶酪凝乳放入奶酪模具中以制作奶酪轮", +  "vintagedelight.jei.info.cheese_wheel": "将4个奶酪凝乳放入奶酪模具中以制作奶酪轮",  
多余空格
 
 
In projects/1.20/assets/vintage-delight/vintagedelight/lang/zh_cn.json:
 &gt; +  "vintagedelight.advancement.harvest_salt": "咸咸的", +  "vintagedelight.advancement.harvest_salt.desc": "收获盐", +  "vintagedelight.advancement.place_fermenting_jar": "腌渍", +  "vintagedelight.advancement.place_fermenting_jar.desc": "放置一个腌制罐", +  "vintagedelight.advancement.place_salt_block": "产盐", +  "vintagedelight.advancement.place_salt_block.desc": "放置在水源附近以产盐.", +  "vintagedelight.advancement.place_evaporator": "蒸发器", +  "vintagedelight.advancement.place_evaporator.desc": "放置一个被水包围的蒸发器，以比盐块快得多的速度产盐", +  "vintagedelight.advancement.get_pickled_item": "在腌制中", +  "vintagedelight.advancement.get_pickled_item.desc": "腌制一次蔬菜", +  "vintagedelight.advancement.obtain_magic_peanut": "魔力花生!", +  "vintagedelight.advancement.obtain_magic_peanut.desc": "也许是一只金鹅?", +  "vintagedelight.advancement.obtain_century_egg": "获得世纪彩蛋", +  "vintagedelight.advancement.obtain_century_egg.desc": "还没有为公布它做好准备", + +  "vintagedelight.advancement.get_jam_item": "粘糊糊'",  ⬇️ Suggested change -  "vintagedelight.advancement.get_jam_item": "粘糊糊'", +  "vintagedelight.advancement.get_jam_item": "黏糊糊",  
原文Jammin'最后的单引号代表省音，是口语中把/ŋ/发成/n/的现象，不保留
 
 
In projects/1.20/assets/vintage-delight/vintagedelight/lang/zh_cn.json:
 &gt; @@ -0,0 +1,186 @@ +{ +  "creativetab.vintage_tab": "腌渍选项卡",  
这个怪怪的（）
 Vintage应当是模组名的一部分，推荐使用模组名或者直接模组名+“选项卡”

**Comment by SieboldiiParlia** [comment]
哦会了会了，抱歉（

File: projects/1.20/assets/vintage-delight/vintagedelight/lang/zh_cn.json [hunk 10]
```diff
@@ -0,0 +1,186 @@
...
+    "item.vintagedelight.chocolate_nut_granola_bar": "巧克力坚果燕麦棒",
+    "item.vintagedelight.fruity_granola_bar": "果味燕麦棒",
+    "item.vintagedelight.deluxe_granola_bar": "豪华燕麦棒",
+    "item.vintagedelight.mason_jar": "梅森罐",
+    "item.vintagedelight.apple_sauce_mason_jar": "苹果汁罐",
+    "item.vintagedelight.apple_sauce_bottle": "苹果汁瓶",
+    "item.vintagedelight.sweet_berry_mason_jar": "甜浆果果酱罐",
+    "item.vintagedelight.sweet_berry_jam_bottle": "甜浆果果酱瓶",
+    "item.vintagedelight.glow_berry_mason_jar": "发光果果酱罐",
+    "item.vintagedelight.glow_berry_jam_bottle": "发光果果酱瓶",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/vintage-delight/vintagedelight/lang/zh_cn.json:113]:
```suggestion
    "item.vintagedelight.glow_berry_mason_jar": "发光浆果果酱罐",
    "item.vintagedelight.glow_berry_jam_bottle": "发光浆果果酱瓶",
```

File: projects/1.20/assets/vintage-delight/vintagedelight/lang/zh_cn.json [hunk 11]
```diff
@@ -0,0 +1,186 @@
...
+  "vintagedelight.advancement.place_fermenting_jar": "腌渍",
+  "vintagedelight.advancement.place_fermenting_jar.desc": "放置一个腌制罐",
+  "vintagedelight.advancement.place_salt_block": "产盐",
+  "vintagedelight.advancement.place_salt_block.desc": "放置在水源附近以产盐",
+  "vintagedelight.advancement.place_evaporator": "蒸发器",
+  "vintagedelight.advancement.place_evaporator.desc": "放置一个被水包围的蒸发器，以比盐块快得多的速度产盐",
+  "vintagedelight.advancement.get_pickled_item": "在腌制中",
+  "vintagedelight.advancement.get_pickled_item.desc": "腌制一次蔬菜",
+  "vintagedelight.advancement.obtain_magic_peanut": "魔力花生!",
+  "vintagedelight.advancement.obtain_magic_peanut.desc": "也许是一只金鹅?",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/vintage-delight/vintagedelight/lang/zh_cn.json:174]:
```suggestion
  "vintagedelight.advancement.obtain_magic_peanut": "魔力花生！",
  "vintagedelight.advancement.obtain_magic_peanut.desc": "也许是一只金鹅？",
```

File: projects/1.20/assets/vintage-delight/vintagedelight/lang/zh_cn.json [hunk 12]
```diff
@@ -0,0 +1,186 @@
...
+  "gui.jei.category.fermenting.time.seconds": "腌制时间",
+  "vintagedelight.jei.info.salt_block": "放置在水源附近以产盐",
+  "vintagedelight.jei.info.evaporator": "放置在水源附近，以比盐块快得多的速度产盐",
+  "vintagedelight.jei.info.organic_compost": "腌制有机醪泥，用作强效肥料，或用于堆肥",
+  "vintagedelight.jei.info.cheese_wheel": "将4个奶酪凝乳放入奶酪模具中以制作奶酪轮",
+
+
+
+  "vintagedelight.advancement.root": "腌渍乐事",
+  "vintagedelight.advancement.root.desc": "来腌制吧!",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/vintage-delight/vintagedelight/lang/zh_cn.json:162]:
```suggestion
  "vintagedelight.advancement.root.desc": "来腌制吧！",
```

**Review by ChuijkYahus — CHANGES_REQUESTED** [review]

**Review by ChuijkYahus — DISMISSED** [review]
丛莓这个名称是怎么来的（）

**Comment by SieboldiiParlia** [comment]
> 丛莓这个名称是怎么来的（）

只是自己突然想的（

**Review by luoxueyasha — COMMENTED** [review]
好像模组更新了？我看bot报多出4行

**Review by luoxueyasha — APPROVED** [review]

**Pull request merged**